backend/src/main/java/edonymyeon/backend/thumbs/application/ThumbsService.java

```diff
@@ -28,21 +29,40 @@ public class ThumbsService {
 
     private final ThumbsRepository thumbsRepository;
 
-    // TODO: 나중에 dev 받아와서 서비스들로 고쳐주기
     private final MemberRepository memberRepository;
+
     private final PostRepository postRepository;
 
+    public AllThumbsInPostResponse findAllThumbsInPost(final Long postId) {
+        AllThumbsInPost allThumbsInPost = new AllThumbsInPost(thumbsRepository.findByPostId(postId));
```

Comment:
```suggestion
        AllThumbsInPost allThumbsInPost = AllThumbsInPost.from(thumbsRepository.findByPostId(postId));
```
정팩메 어떠신가요? new로 생성하니까 엔티티 같네요🥺